### PR TITLE
docs: add screenshot requirement to PR template

### LINKED ISSUE #198: docs: Add screenshot requirement to PR template
## Description

We received a handfull of PRs that missed the step of generating profile and badge on the README.

## Proposed Solution

Add a section of "Screenshot" after the "Added to documentation?" to the PR template. There will be an explanation here that contributors need to provide a screenshot of their profile being generated on README. This will ease maintainers in reviewing PRs and ensure contributors to run the generate command.

### PR DESCRIPTION
**Pull request by CBID2**
## Description
Lately, there has been a handful of Pull Requests that missed the step of generating profile and badge on the README. So, @adiati98, @BekahHW, and I agreed that the PR template needs a section called "Screenshot" with instructions after the "Added to documentation?" section.  By providing this information, maintainers would have an easier time reviewing PRs and ensuring that contributors followed the steps to adding their profile.

## Issue Addressed
Closes #198 

## What type of PR is this? (check all applicable)

- [ ] 🤝 Add a contributor
- [x] 📝 Documentation Update

## Contributors checklist (check all applicable)

- Have you read the [Getting Started](https://github.com/open-sauced/guestbook#getting-started) section thoroughly?

  - [ ] ✅ Yes
  - [ ] ❌ Not yet
  - [x] 🚫 Not applicable 

- How did you add yourself as a contributor?

  - [ ] 🤖 With CLI
  - [ ] ⌨️ Manually
  - [x] 🚫 Not applicable 

    If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?

    - [ ] ✅ Yes
    - [ ] ❌ No
    - [x] 🚫 Not applicable 

- Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
  - [ ] ✅ Yes
  - [x] ❌ No

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @adiati98! :) I made the PR.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ This PR adds <github-username> as a contributor.
 - [ ] 📜 README.md
 - [ ] 🙅 no documentation needed
 
+## Screenshot
+
+<!-- Please provide a screenshot of your profile being generated on the README. This will make it easier for maintainers when reviewing PRs and ensure that you to ran the generate command as mentioned in the previous question. -->
```
Comment (adiati98) [inline .github/PULL_REQUEST_TEMPLATE.md:41]:
```suggestion
<!-- Please provide a screenshot of your profile being generated on the README. This ensures that you run the `npm run contributors:generate` command, as mentioned in the previous question, making it easier for maintainers to review PRs. -->
```

**Comment by CBID2** [comment]
Hi @adiati98! :) I made the change! :)

**Review by adiati98 — APPROVED** [review]
Thanks, @CBID2!

**Pull request merged**